### DISCUSSION TITLE
Un seul descendant ?

### DISCUSSION THREAD

**Discussion by julzKB** [FreshRSS/FreshRSS#5023] [Q&A] [answered]
Juste une petite question pour m'orienter dans la bonne direction .

Je crée mon flux en scrapant une page, mais je n'obtiens toujours qu`un seul article "descendant" au lieu de plusieurs en utilisant un testeur de Xpath, la page est pourtant simple, est-ce que quelque chose vous viendrait en tête immediatement ?

voici l'url de la page au cas ou :

https://southernlord.com/product-category/departments/southern-lord-releases/

<img width="705" alt="Screenshot 2023-01-20 at 5 44 32 PM" src="https://user-images.githubusercontent.com/831656/213634146-4791c7a2-1691-4664-ae7c-bba7f89f914c.png">

**Comment by Alkarex** [comment, marked as answer]
Bonjour,
Votre sélecteur `//ul[@class="..."]` retourne un seul élément qui est le parent des articles. Or ce sélecteur doit retourner une liste de tous les articles, pas juste le parent.

Essayez plutôt avec `//h4/ancestor::li` (par exemple) pour trouver les articles.

![image](https://user-images.githubusercontent.com/1008324/213640978-8b89737f-b28e-43ad-92b1-c4ac6a1d18ca.png)

**Comment by julzKB** [comment]
en effet ! merci pour le coup de main !
